enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -16,6 +16,7 @@ const units = (px: number) => ({
 
 export const FONT_SIZES = {
```

Comment:
Not related to the current change, but this constant isn't used anyway as far as I can tell. Can we remove `export` from it?

Reply:
Nice, didn't notice that before - will do!